utilities/constants.py

```diff
@@ -109,6 +112,19 @@ class StorageClassName:
     NFS: str = "nfs"
 
 
+class Dashboard:
+    route_name = "odh-dashboard" if py_config.get("distribution") == "upstream" else "rhods-dashboard"
+    label = OPENDATAHUB_IO + "/dashboard"
```

Comment:
arg name should represent what it contains; please place under labels class

Reply:
done